webrtc/src/main/resources/static/client.js

```diff
@@ -0,0 +1,116 @@
+//connecting to our signaling server 
+var conn = new WebSocket('ws://localhost:8080/socket');
+
+conn.onopen = function() {
+	console.log("Connected to the signaling server");
```

Comment:
Use spaces instead of tabs for indentation.

Reply:
I have done this change.